packages/tre/src/workers/core/entry.worker.ts

```diff
@@ -0,0 +1,205 @@
+// @ts-expect-error "devalue" is ESM-only, so TypeScript requires us to use the
+//  `*.mts` extension, or set `"type": "module"` in our `package.json`. We can't
+//  do the first as ambient types from `@cloudflare/workers-types` don't seem to
+//  work, and the second would affect all consumers of `miniflare`.
+import { unflatten } from "devalue";
```

Comment:
https://devblogs.microsoft.com/typescript/announcing-typescript-5-0-beta/#moduleresolution-bundler should fix this if the Typescript version can be upgraded to 5?

Reply:
What would that look like? What's the benefit of api extractor over that approach?